pg_temp.py

```diff
@@ -21,6 +22,10 @@
 temp_db = None
 
 
+def flatten(listOfLists):
+    return itertools.chain.from_iterable(listOfLists)
```

Comment:
Small sug -- avoid `camelCase` names -- maybe `values` here instead of `listsOfLists`?  Since this function is just a one-liner we could just call `from_iterable` at the call site instead of needing the function too.

Reply:
I copied this code from python's documentation for itertools
https://docs.python.org/3.7/library/itertools.html#itertools-recipes